charts/openmetadata/values.yaml

```diff
@@ -254,7 +262,7 @@ openmetadata:
         secretKey: ""
     secretsManager:
       enabled: true
-      # Possible values are db, aws, aws-ssm, managed-aws, managed-aws-ssm, in-memory, managed-azure-kv, azure-kv
+      # Possible values are db, aws, aws-ssm, managed-aws, managed-aws-ssm, in-memory, managed-azure-kv, azure-kv, gcp
```

Comment:
are the list of values informed anywhere else to provide what is really supported or not? Can I use a helm value as `potato` and it will eat it?

Reply:
You can use `potato` in every string field, we are not doing enum validations. A nice to have feature though :)